### PR TITLE
Create homepage Announcements component

### PR DESCRIPTION
**Pull request by lilyndk**
## Description
- Add 'Announcements' content and 'card large' styles
- Add second row bg color option (light gray)
- Fix secondary btn border override
- Adjust font sizes to align more with Figma designs (buttons, homepage h2)

## Change Type
- [ ] Bug Fix
- [X] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Related Issue
(https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/146)

## Screenshots
screencapture-127-0-0-1-4000-2024-05-01-17_12_42

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [X] I have tested my changes locally.
- [X] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _sass/main.scss [hunk 1]
```diff
@@ -3,14 +3,16 @@
 @import 'bootstrap/bootstrap';
 @import 'custom/custom';
 
-$backgroundColor: #fff;
```
Comment (lilyndk) [inline _sass/main.scss:6]: not common to create white bg color variable

File: _sass/custom/_home.scss [hunk 2]
```diff
@@ -60,22 +60,29 @@
...
+                max-width: fit-content;
+            }
+            .card-body {
+                padding: 2rem 2rem 3rem 2rem;
+                align-items: center;
+            }
         }
     }
 
-    .row.mentors-cta {
```
Comment (lilyndk) [inline _sass/custom/_home.scss:72]: moved these card styles to '_card.scss' file.

File: _sass/custom/_card.scss [hunk 3]
```diff
@@ -1,11 +1,12 @@
 .card {
     border-radius: 1rem;
+    height: 100%;
+}
+.card-date {
```
Comment (dricazenck) [inline _sass/custom/_card.scss:6]:
```suggestion

.card-date {
```

File: _sass/custom/_card.scss [hunk 4]
```diff
@@ -55,6 +53,60 @@
...
+    }
+    .card-media {
+        padding: 0;
+    }
+    img {
+        max-width: 100%;
+        max-height: 100%;
+    }
+}
+@include media-breakpoint-up(md) {
```
Comment (dricazenck) [inline _sass/custom/_card.scss:74]:
```suggestion

@include media-breakpoint-up(md) {
```

File: _sass/custom/_card.scss [hunk 5]
```diff
@@ -55,6 +53,60 @@
...
+        }
+        img {
+            border-radius: 4px;
+        }
+    }
+    .card-body {
+        padding: 0 0 1rem 0;
+    }
+}
+@include media-breakpoint-up(md) {
```
Comment (dricazenck) [inline _sass/custom/_card.scss:105]:
```suggestion

@include media-breakpoint-up(md) {
```

File: _sass/main.scss [hunk 6]
```diff
@@ -28,6 +30,13 @@ body {
 .row {
   justify-content: center;
 
+  &.bg-1 {
+    background-color: $bg-1;
+  }
+  &.bg-2 {
```
Comment (dricazenck) [inline _sass/main.scss:37]:
```suggestion

  &.bg-2 {
```

File: _sass/main.scss [hunk 7]
```diff
@@ -3,14 +3,16 @@
...
+$bg-2: #f7f7f7;
+
 $bodyColor: #000000;
 $bodyFont: 'Roboto', sans-serif;
 
 $container-max-width: 1376px;
 
 body {
-  background: $backgroundColor;
+  background: #fff;
```
Comment (dricazenck) [inline _sass/main.scss:15]:
```suggestion
  background: $white;
```

**Review by dricazenck — APPROVED** [review]
Thank you for the quick solution!! you 🚀

**Pull request merged**